### PR TITLE
[US163 Task 173] Move Taiga Story APIs

### PR DESCRIPTION
**Pull request by sverma89asu**

### CODE REVIEW

File: taigaProject/Dockerfile [hunk 1]
```diff
@@ -14,4 +14,4 @@ RUN pip install --no-cache-dir --upgrade -r ./requirements.txt
 COPY . /code/taigaProject
 WORKDIR taigaProject/src
 EXPOSE 8000
-CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000"]
+CMD ["uvicorn", "main:app", "--reload", "--host", "0.0.0.0", "--port", "8000"]
```
Comment (achaur11) [inline taigaProject/Dockerfile]: Why are you adding reload over here?

**Comment by shahih25** [comment]
Can you please move **get_user_story_details** as well?

File: taigaProject/taiga/userStory/get_user_story_history.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        return []
+
+
+def get_user_story_details_by_id(
+    story,
+    headers,
+    taiga_url
+):
+    task_history_url = f"{taiga_url}/history/userstory/{story['id']}"
+    return requests.get(task_history_url, headers=headers)
```
Comment (shahih25) [inline taigaProject/taiga/userStory/get_user_story_history.py:74]: please add try-catch block in the next pr.

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**